ts/screens/authentication/IdpLoginScreen.tsx

```diff
@@ -112,12 +120,28 @@ class IdpLoginScreen extends React.Component<Props, State> {
     this.props.dispatchLoginFailure(
       new Error(`login failure with code ${errorCode || "n/a"}`)
     );
+    instabugLog(
```

Comment:
```suggestion
    instabugLog(
      fromNullable(errorCode).fold(
        "login failed with no error code available",
        ec => `login failed with code (${ec}) : ${getSpidDescription(ec)}`
      ),
      TypeLogs.ERROR,
      "login"
    );
```
pretty the same but more readable

Reply:
changed in 417854f, but for my personal taste I think it's almost less readable.

Reply:
@fabriziofff 
I think it's pretty the same until there is only one ternary operator (such in this case).
imho the code becomes ugly when there are nested > 1 of them

(anyway that is just a suggestion not an enforcement)